lib/import/admin.js

```diff
@@ -129,6 +136,7 @@ function save_features(feature_collection) {
     feat.name = feat.properties.name;
     feat.admin_code = feat.properties.admin_code;
     feat.geo_area_sqkm = feat.properties.geo_area_sqkm;
+    feat.population = feat.properties.population;
```

Comment:
Again, check if this value exists.

Maybe the schema should have a validator that this field is a number..